src/renderer/helpers/walletHelper.test.ts

```diff
@@ -4,31 +4,45 @@ import * as NEA from 'fp-ts/lib/NonEmptyArray'
 import * as O from 'fp-ts/lib/Option'
 
 import { ASSETS_TESTNET } from '../../shared/mock/assets'
+import { WalletType } from '../../shared/wallet/types'
 import { NonEmptyWalletBalances } from '../services/wallet/types'
 import { eqWalletBalances } from './fp/eq'
 import {
   filterWalletBalancesByAssets,
   getAssetAmountByAsset,
   getBnbAmountFromBalances,
   getLtcAmountFromBalances,
-  getWalletBalanceByAsset
+  getWalletBalanceByAsset,
+  getWalletByAddress
 } from './walletHelper'
 
 describe('walletHelper', () => {
   const RUNE_WB = {
     amount: baseAmount('12300000000'),
     asset: AssetRuneNative,
-    walletAddress: 'rune native wallet address'
+    walletAddress: 'rune native wallet address',
+    walletType: 'keystore' as WalletType
```

Comment:
Try `const RUNE_WB: RUNE_WB` -> No need to use `as ...` for `WalletType`. Same for all other `XYZ_WB`